src/snapred/backend/data/GroceryService.py

```diff
@@ -134,14 +136,27 @@ def _createNeutronFilename(self, runNumber: str, useLiteMode: bool) -> str:
         ext = instr + ".extension"
         return self.getIPTS(runNumber) + Config[pre] + str(runNumber) + Config[ext]
 
-    def _createGroupingFilename(self, groupingScheme: str, useLiteMode: bool) -> str:
+    def _createGroupingFilename(self, runNumber: str, groupingScheme: str, useLiteMode: bool) -> str:
         if groupingScheme == "Lite":
-            return str(Config["instrument.lite.map.file"])
-        instr = "lite" if useLiteMode else "native"
-        home = "instrument.calibration.powder.grouping.home"
-        pre = "grouping.filename.prefix"
-        ext = "grouping.filename." + instr + ".extension"
-        return f"{Config[home]}/{Config[pre]}{groupingScheme}{Config[ext]}"
+            path = str(Config["instrument.lite.map.file"])
+        elif self._groupingMapCache.get(runNumber):
```

Comment:
it may be unnecessary to cache this?  We may face similar issues to caching calibration jsons.

Reply:
I'll remove the caching